meta-mender-raspberrypi-demo/classes/rpi-demo-wifi.bbclass

```diff
@@ -1,7 +1,7 @@
 # Global variables
 DISTRO_FEATURES_append = " wifi"
 
-IMAGE_INSTALL_append = " iw wpa-supplicant linux-firmware-raspbian-bcm43430 packagegroup-base"
```

Comment:
The fix is probably to rename:

    linux-firmware-raspbian-bcm43430

to

    linux-firmware-bcm43430

Reply:
Updated.  Thank you.